trino/sqlalchemy/compiler.py

```diff
@@ -10,6 +10,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 from sqlalchemy.sql import compiler
+try:
+    from sqlalchemy.sql.expression import (
+        Alias,
+        CTE,
+        Subquery,
+    )
+except ImportError:
+    from sqlalchemy.sql.expression import Alias
+    CTE = type(None)
+    Subquery = type(None)
```

Comment:
Can you leave a comment here why this is ok to do?

Reply:
Yep, this is because the `CTE` and `Subquery` classes weren't introduced until SQLAlchemy 1.4. Added a comment.